easybuild/easyconfigs/c/CP2K/CP2K-2.4.0_ipi.patch

```diff
@@ -0,0 +1,510 @@
+#This patch makes CP2K source code compatible with i-pi
```

Comment:
@boegel: This is a patch the i-pi developers created. It can be found here:
https://github.com/i-pi/i-pi/tree/master/patches

In the documentation it is explained that i-PI is distributed with patch files. When
applied to a clean distribution, these will adjust the source code to make it compatible
with i-PI. So, maybe I should modify the patch file and add a link from where I got the patch.